pages.pt_BR/common/aws-cur.md

```diff
@@ -0,0 +1,16 @@
+# aws cur
+
+> Cria, pesquisa e apaga relatórios de uso do AWS.
+> Mais informações: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/cur/index.html>.
+
+- Cria um relatório de uso e custo do AWS definido de a partir de um arquivo JSON:
+
+`aws cur put-report-definition --report-definition file://{{caminho/para/definição-do-relatório.json}}`
```

Comment:
We use `snake_case` for placeholders
```suggestion
`aws cur put-report-definition --report-definition file://{{caminho/para/definição_do_relatório.json}}`
```

Reply:
@caduvieira Oh, I see this issue is in the main (English) page too. Could you edit that page inside this PR?